.travis.yml

```diff
@@ -11,8 +11,8 @@ before_install:
 
 install:
   - bundle install
-  - bundle exec pod repo update
-  - bundle exec pod install
+  - bundle exec pod repo update --silent
```

Comment:
The log is too long and wasteful so added this option, but if you consider it is necessary, I will revert https://github.com/braintree/braintree_ios/pull/303/commits/8ec8cab9feb9b441a7c4b336a92309cb8a8c55db.